HappyAnding/HappyAnding/Views/Components/ListShortcutView.swift

```diff
@@ -8,83 +8,78 @@
 import SwiftUI
 
 /// - parameters:
-/// - categoryName: 카테고리에서 접근할 시, 해당 카테고리의 이름을 넣어주시고, 그렇지 않다면 nil을 넣어주세요
 /// sectionType: 다운로드 순위에서 접근할 시, .download를, 사랑받는 앱에서 접근시 .popular를 넣어주세요.
 struct ListShortcutView: View {
     
     @EnvironmentObject var shortcutsZipViewModel: ShortcutsZipViewModel
     
-    @State var shortcuts:[Shortcuts]?
-//    @State var shortcutsArray: [Shortcuts] = []
+    @State var data: NavigationListShortcutType
+    @State var shortcutsArray: [Shortcuts] = []
     @State private var isLastItem = false
-    @State var description: String = ""
-    
-    // TODO: let으로 변경필요, 현재 작업중인 코드들과 충돌될 가능성이 있어 우선 변수로 선언
-    var categoryName: Category?
-    var sectionType: SectionType?
     
     var body: some View {
-        
-        List {
-            
-            if sectionType != .myShortcut {
-                header
-                    .listRowBackground(Color.Background)
-                    .listRowSeparator(.hidden)
-                    .listRowInsets(EdgeInsets())
-            }
-            
-            //TODO: 무한 스크롤을 위한 업데이트 함수 필요
-            if let shortcuts {
-                ForEach(Array(shortcuts.enumerated()), id: \.offset) { index, shortcut in
-                    if sectionType == .download {
-                        ShortcutCell(shortcut: shortcut, rankNumber: index + 1)
-                            .listRowInsets(EdgeInsets())
-                            .listRowSeparator(.hidden)
+        ScrollView {
+            LazyVStack {
```

Comment:
```suggestion
            LazyVStack(spacing: 0) {
```
기본 여백이 존재해서 여백이 커진 것 같습니다~!